package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "grunt pre-mocha && node test-runner",
     "update-i18n": "node script/i18n.mjs",
-    "prepare": "bower update && grunt deploy-build"
+    "prepare": "grunt prepare"
```

Comment:
This break chart and graph regarding the use of the side-by-side-improve used to generate the bar chart